app.css

```diff
@@ -172,7 +172,8 @@ body {
       font-size: .9em;
       white-space: nowrap;
       overflow: hidden;
-      text-overflow: ellipsis; }
```

Comment:
It's not documented, my bad, but style modification have to be made in the `app.scss`  source file and then compiled with for exemple:
```sh
$ sassc app.scss > app.css
```
It's undocumented, my bad (because I'm supposed to add a better way to do that at some point :p)

Reply:
done :)